Update java9.options file to include jdk.attach/sun.tools.attach for Dump action

When using Java16 Hotspot, `java.lang.IllegalAccessException` occurs, when trying to create thread dumps. Need to add the `--add-exports jdk.attach/sun.tools.attach=ALL-UNNAMED` globally in the java9.options file, so the runtime is not affected.

```
> Stack Dump = java.lang.IllegalStateException: java.lang.IllegalAccessException: class com.ibm.ws.kernel.boot.internal.commands.HotSpotJavaDumperImpl$VirtualMachine cannot access class sun.tools.attach.HotSpotVirtualMachine (in module jdk.attach) because module jdk.attach does not export sun.tools.attach to unnamed module @7dc5e7b4
> at com.ibm.ws.kernel.boot.internal.commands.HotSpotJavaDumperImpl$VirtualMachine.remoteDataDump(HotSpotJavaDumperImpl.java:341)
> at com.ibm.ws.kernel.boot.internal.commands.HotSpotJavaDumperImpl.createThreadDump(HotSpotJavaDumperImpl.java:195)
> at com.ibm.ws.kernel.boot.internal.commands.HotSpotJavaDumperImpl.dump(HotSpotJavaDumperImpl.java:101)
> at com.ibm.ws.kernel.launch.internal.FrameworkManager.dumpJava(FrameworkManager.java:1233)
> at com.ibm.ws.kernel.launch.internal.FrameworkManager.dumpJava(FrameworkManager.java:1219)
> at com.ibm.ws.kernel.launch.internal.LibertyProcessImpl.createJavaDump(LibertyProcessImpl.java:106)
> at com.ibm.ws.request.timing.manager.ThreadDumpManager$1.run(ThreadDumpManager.java:108)
> at com.ibm.ws.threading.internal.SchedulingRunnableFixedHelper.run(SchedulingRunnableFixedHelper.java:290)
> at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
> at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
> at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:238)
> at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
> at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
> at java.base/java.lang.Thread.run(Thread.java:831)
>Caused by: java.lang.IllegalAccessException: class com.ibm.ws.kernel.boot.internal.commands.HotSpotJavaDumperImpl$VirtualMachine cannot access class sun.tools.attach.HotSpotVirtualMachine (in module jdk.attach) because module jdk.attach does not export sun.tools.attach to unnamed module @7dc5e7b4
> at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:385)
> at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:687)
> at com.ibm.ws.kernel.boot.internal.commands.HotSpotJavaDumperImpl$VirtualMachine.remoteDataDump(HotSpotJavaDumperImpl.java:339)
```

--- a/dev/com.ibm.ws.kernel.boot/publish/platform/java/java9.options
+++ b/dev/com.ibm.ws.kernel.boot/publish/platform/java/java9.options
@@ -10,6 +10,9 @@ java.naming/com.sun.jndi.url.ldap=ALL-UNNAMED
 # for Oracle Kerberos
 --add-exports
 java.security.jgss/sun.security.krb5.internal=ALL-UNNAMED
+# for Dump action: com.ibm.ws.kernel.boot.internal.commands.HotSpotJavaDumperImpl$VirtualMachine.remoteDataDump(HotSpotJavaDumperImpl.java:339)
+--add-exports
+jdk.attach/sun.tools.attach=ALL-UNNAMED
 ### Opens
 --add-opens
 java.base/java.util=ALL-UNNAMED

--- a/dev/com.ibm.ws.request.timing.hung_fat/fat/src/com/ibm/ws/request/timing/hung/fat/HungRequestEnableThreadDumps.java
+++ b/dev/com.ibm.ws.request.timing.hung_fat/fat/src/com/ibm/ws/request/timing/hung/fat/HungRequestEnableThreadDumps.java
@@ -34,7 +34,6 @@ import componenttest.annotation.Server;
 import componenttest.custom.junit.runner.FATRunner;
 import componenttest.custom.junit.runner.Mode;
 import componenttest.custom.junit.runner.Mode.TestMode;
-import componenttest.topology.impl.JavaInfo;
 import componenttest.topology.impl.LibertyServer;
 
 /**
@@ -53,13 +52,7 @@ public class HungRequestEnableThreadDumps {
 
     @BeforeClass
     public static void setUp() throws Exception {
-        JavaInfo java = JavaInfo.forCurrentVM();
         ShrinkHelper.defaultDropinApp(server, "TestWebApp", "com.ibm.testwebapp");
-        int javaVersion = java.majorVersion();
-        if (javaVersion != 8) {
-            CommonTasks.writeLogMsg(Level.INFO, " Java version = " + javaVersion + " - It is higher than 8, adding --add-exports...");
-            server.copyFileToLibertyServerRoot("add-exports/jvm.options");
-        }
         CommonTasks.writeLogMsg(Level.INFO, " Starting server...");
         server.startServer();
     }

--- a/dev/com.ibm.ws.request.timing.hung_fat/fat/src/com/ibm/ws/request/timing/hung/fat/HungRequestTiming.java
+++ b/dev/com.ibm.ws.request.timing.hung_fat/fat/src/com/ibm/ws/request/timing/hung/fat/HungRequestTiming.java
@@ -47,7 +47,6 @@ import componenttest.annotation.Server;
 import componenttest.custom.junit.runner.FATRunner;
 import componenttest.custom.junit.runner.Mode;
 import componenttest.custom.junit.runner.Mode.TestMode;
-import componenttest.topology.impl.JavaInfo;
 import componenttest.topology.impl.LibertyServer;
 
 @RunWith(FATRunner.class)
@@ -65,13 +64,7 @@ public class HungRequestTiming {
 
     @BeforeClass
     public static void setUp() throws Exception {
-        JavaInfo java = JavaInfo.forCurrentVM();
         ShrinkHelper.defaultDropinApp(server, "TestWebApp", "com.ibm.testwebapp");
-        int javaVersion = java.majorVersion();
-        if (javaVersion != 8) {
-            CommonTasks.writeLogMsg(Level.INFO, " Java version = " + javaVersion + " - It is higher than 8, adding --add-exports...");
-            server.copyFileToLibertyServerRoot("add-exports/jvm.options");
-        }
         CommonTasks.writeLogMsg(Level.INFO, " starting server...");
         server.startServer();
     }

--- a/dev/com.ibm.ws.request.timing.hung_fat/publish/files/add-exports/jvm.options
+++ b/dev/com.ibm.ws.request.timing.hung_fat/publish/files/add-exports/jvm.options
@@ -1,3 +1,0 @@
-# for dump action: com.ibm.ws.kernel.boot.internal.commands.HotSpotJavaDumperImpl$VirtualMachine.remoteDataDump(HotSpotJavaDumperImpl.java:342)
---add-exports
-jdk.attach/sun.tools.attach=ALL-UNNAMED

--- a/dev/com.ibm.ws.request.timing_fat/fat/src/com/ibm/ws/request/timing/fat/TimingRequestTiming.java
+++ b/dev/com.ibm.ws.request.timing_fat/fat/src/com/ibm/ws/request/timing/fat/TimingRequestTiming.java
@@ -39,7 +39,6 @@ import componenttest.annotation.Server;
 import componenttest.custom.junit.runner.FATRunner;
 import componenttest.custom.junit.runner.Mode;
 import componenttest.custom.junit.runner.Mode.TestMode;
-import componenttest.topology.impl.JavaInfo;
 import componenttest.topology.impl.LibertyServer;
 
 @RunWith(FATRunner.class)
@@ -61,13 +60,7 @@ public class TimingRequestTiming {
 
     @BeforeClass
     public static void setUp() throws Exception {
-        JavaInfo java = JavaInfo.forCurrentVM();
         ShrinkHelper.defaultDropinApp(server, "jdbcTestPrj_3", "com.ibm.ws.request.timing");
-        int javaVersion = java.majorVersion();
-        if (javaVersion != 8) {
-            CommonTasks.writeLogMsg(Level.INFO, " Java version = " + javaVersion + " - It is higher than 8, adding --add-exports...");
-            server.copyFileToLibertyServerRoot("add-exports/jvm.options");
-        }
         CommonTasks.writeLogMsg(Level.INFO, " starting server...");
         server.startServer();
     }

--- a/dev/com.ibm.ws.request.timing_fat/publish/files/add-exports/jvm.options
+++ b/dev/com.ibm.ws.request.timing_fat/publish/files/add-exports/jvm.options
@@ -1,3 +1,0 @@
-# for dump action: com.ibm.ws.kernel.boot.internal.commands.HotSpotJavaDumperImpl$VirtualMachine.remoteDataDump(HotSpotJavaDumperImpl.java:342)
---add-exports
-jdk.attach/sun.tools.attach=ALL-UNNAMED

